scopes/envs/envs/envs.cmd.tsx

```diff
@@ -31,19 +31,19 @@ type GetEnvOpts = {
 
 export class GetEnvCmd implements Command {
   name = 'get <component-name>';
-  description = "show information about a component's env";
+  description = "show config information from a component's env";
   arguments = [
     {
       name: 'component-name',
-      description: "the 'component name' or 'component id' of the component its env you'd like to inspect",
+      description: "the 'component name' or 'component id' of the component whose env you'd like to inspect",
     },
   ];
-  examples: [{ cmd: 'get ui/button'; description: 'show information about the env configured for ui/button' }];
+  examples: [{ cmd: 'get ui/button'; description: 'show config information from the env configured for ui/button' }];
   options = [
     [
       '',
       'services <string>',
-      'show information about the specific services only. for multiple services, separate by a comma and wrap with quotes',
+      'show information about the specific services only. for multiple services, separate by a comma and wrap with quotes', // @david do we provide a list anywhere for what values can be used here?
```

Comment:
we probably need another command for this or something, as this is defined in runtime by various harmony aspects, so there's no definitive list.

Reply:
so something like the tasks list you get from bit build --list-tasks?

Reply:
i think so. but this is a different discussion and out of the scope of this PR.

Reply:
agreed
but something we should create a task for on the back of this PR